docs/self-managed/platform-deployment/helm-kubernetes/platforms/amazon-eks/dual-region.md

```diff
@@ -0,0 +1,603 @@
+---
+id: dual-region
+title: "Dual-region setup"
+description: "Deploy two Amazon Kubernetes (EKS) clusters with Terraform for a peered setup allowing dual-region communication."
+---
+
+import CoreDNSKubeDNS from "./assets/core-dns-kube-dns.svg"
+
+:::warning
+Review our [dual-region concept documentation](#) <!-- TOOD: link reference --> before continuing to understand the current limitations and restrictions of this setup, as well as the disclaimer concerning support from Camunda.
```

Comment:
Looks like we still need to link reference here?

Reply:
thank you :bow: ,fixed.
I should write myself a script for checking for all kind of typos of `TODO` 😅